src/main/java/bio/terra/cli/context/WorkspaceContext.java

```diff
@@ -30,16 +30,16 @@
   // workspace description object returned by WSM
   public WorkspaceDescription terraWorkspaceModel;
 
-  // map of cloud resources for this workspace (name -> object)
-  public Map<String, CloudResource> cloudResources;
+  // map of cloud resources for this workspace (name -> WSM resource description object)
+  public Map<String, ResourceDescription> resources;
```

Comment:
Is this a breaking change? Are existing CLI workspace files going to be incompatible? I think it's OK for us to make breaking changes at this point, as long as we're coordinating with the solutions team and know exactly what the impact is.
Not this PR, but this starts to feel like we might want to think about data serialization formats for the CLI @jmczerk.

Reply:
Yep it's a breaking change in many ways: reorganized commands, changed the context serialization format, resources previously created will be unreachable. Of all the changes, the reorganized commands may be the largest burden, because Nicole's notebook scripts will be broken.

Certainly interested to hear any serialization advice you have!

Reply:
Probably a good discussion to start having out side the PR...  It seems like a very bounded use case which should be very amenable to evolving schemas without a lot of overhead.  I assume human readability/edit-ability is desirable?

Reply:
Yes, I think human readable is always preferable, unless there's a good reason against. I don't expect the context we persist on disk to grow extremely large -- we're reading it all into memory now, so if we do ever get very large workspaces in terms of the number of resources, that would require much larger code changes anyway.